include/event.php

```diff
@@ -516,9 +535,9 @@ function event_by_id($owner_uid = 0, $event_params, $sql_extra = '') {
 		intval($event_params["event_id"])
 	);
 
-	if (dbm::is_result($r))
-		return $r;
-
+	if (dbm::is_result($r)) {
+		return event_remove_duplicates($r);
```

Comment:
Done. Maybe we can change this before the release. But I would like to concentrate now on the other issues.